src/abstractdataframe/selection.jl

```diff
@@ -596,6 +584,12 @@ the same parent data frame they might get corrupt.
 
 $TRANSFORMATION_COMMON_RULES
 
+There the following keyword arguments are supported by `select!`:
+- `renamecols` : whether in the `cols => function` form automatically generated
```

Comment:
I think in general the convention is to repeat the argument type and default value. That's especially useful here because the signature is separated from this section by a large block of text.

Reply:
ok - fixed